blockchain/projects.go

```diff
@@ -187,11 +172,15 @@ func (p *Projects) CreateInitialAccounts(projectID uuid.UUID) ([]*model.Account,
 		if err != nil {
 			return nil, err
 		}
+	}
 
-		account := model.AccountFromFlow(flowAccount, projectID)
-		account.Index = i
-		account.ID = uuid.New()
-		accounts[i] = account
+	var accounts []*model.Account
+	for i := 0; i < p.accountsNumber; i++ {
```

Comment:
couldn't you just collect the accounts from `createAccount` method?

Reply:
We still need to use p.getAccount() since there is logic there to obtain the project state from memory as well. However, nice catch that there's no need for a second loop!